Cargo.toml

```diff
@@ -17,7 +17,7 @@ members = [
   "trace-utils",
   "spawn_worker",
   "tests/spawn_from_lib",
-  "serverless"
+  "metrics",
```

Comment:
I'm curious -- is this really intended (e.g. remove the "serverless" folder?) Doesn't that mean it will stop getting built and tested in CI?

Reply:
I believe we do want to keep the serverless folder. I changed it back